Cargo.toml

```diff
@@ -1,65 +1,41 @@
-[package]
-name = "osmosis-liquidity-helper"
-version = "0.1.0"
-authors = ["Sturdy <sturdy@apollo.farm>"]
-edition = "2021"
-
-exclude = [
-  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
-  "contract.wasm",
-  "hash.txt",
-]
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[lib]
-crate-type = ["cdylib", "rlib"]
+[workspace]
+members = ["contracts/*", "packages/*"]
+
+[workspace.package]
+authors       = ["Sturdy <sturdy@apollo.farm>"]
+edition       = "2021"
+license       = "MPL-2.0"
+homepage      = "https://apollo.farm"
+repository    = ""
+documentation = ""
+keywords      = ["blockchain", "cosmos", "cosmwasm"]
+rust-version  = "1.64.0"
+
+[workspace.dependencies]
+cosmwasm-schema  = "1.1.8"
+cosmwasm-std     = "1.1.8"
+cw2              = "1.0.0"
+cw20             = "1.0.0"
+cw-storage-plus  = "0.16.0"
+thiserror        = "1.0.31"
+schemars         = "0.8.10"
+serde            = { version = "1.0.145", default-features = false, features = ["derive"] }
+cw-asset         = { git = "https://github.com/apollodao/cw-asset", rev = "5e51773da42ef82faa67576454e8c4b0971097f2" }
+cw-dex           = { git = "https://github.com/apollodao/cw-dex", rev = "19d3c9be9edc8ca48cdeb02c4deecc0b7daed813" }
+apollo-utils     = { git = "https://github.com/apollodao/apollo-utils", rev = "0421a13a2b7cbbce76ea88563a9ef4fce3d1e6ee" }
+liquidity-helper = { path = "./packages/liquidity-helper" }
+
+# Dev dependencies
+osmosis-testing = { git = "https://github.com/apollodao/osmosis-rust.git", rev = "68281423c7a19b002e217a7a248bee11e2ab6436" }
+cw-it           = { git = "https://github.com/apollodao/cw-it", rev = "35406c86b0bfb6e0f70df22d7ab3690a7c7ab59d" }
+test-case       = "2.2.2"
```

Comment:
does this just set the version but leaving it so that it is only compiled for the tests since these are only used as dev deps in the contracts?

Reply:
I think (and hope) so. Can try to double check.

Reply:
![Screenshot 2022-12-14 at 13 50 57](https://user-images.githubusercontent.com/91910406/207599827-7387cb75-318b-48e5-9cbd-54a4f1b0ad9f.png)
Yes, it seems e.g. `test-case` is only included in dev-dependencies (output from `cargo-tree`).